lib/gollum/app.rb

```diff
@@ -188,7 +188,7 @@ class App < Sinatra::Base
               @etag         = page.sha
               mustache :edit
           else
-            redirect_to("/create/#{encodeURIComponent(@name)}")
```

Comment:
I removed `/create` because gollum tries to create a `no-exist` file **within the `/create` directory** when it is redirected to `/create/no-exist`. Redirecting to `/no-exist` lets gollum do the job. Is it possible to follow two-step redirection in the test?

https://github.com/gollum/gollum/blob/edc7d0b50bdb4a6477ad1bccef7651af293d2e94/test/test_app.rb#L304-L310

Reply:
My bad. Redirecting to `/gollun/create/no-exist` is the way to go.

Reply:
Understood. Are you managing to create a test guarding against the, I assume, bug with base-path?